server/api/slack.py

```diff
@@ -51,8 +51,9 @@ def notify_user(message, slack_id):
     """
     return slack_client.api_call(
         "chat.postMessage",
-        channel=slack_id,
+        channel="#general",
```

Comment:
Should this go to general or to a specifc channel, you can create a slack channel for this though.

Reply:
True,it should go to a specific channel.Do you have any suggestions for the name of the channel?

Reply:
@dkam26 you can create a channel called **a_social_events**

Reply:
You could use a bot to send dm’s to the user as a notification.
which to me is more precise and less intrusive

Reply:
I agree. It would be less intrusive.

Reply:
The initial IM channel which is the target user's Slack ID is actually what we want. I am not sure we need this change @dkam26